### PR TITLE
[REVIEW] Map multiple netcdf vars to one gridadmin field

### PR DESCRIPTION
**Pull request by jpprins1**

### CODE REVIEW

File: threedigrid/admin/nodes/timeseries_mixin.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
 class NodeResultsMixin(ResultMixin):
     s1 = TimeSeriesArrayField()
     su = TimeSeriesArrayField()
     vol = TimeSeriesArrayField()
     rain = TimeSeriesArrayField()
+    ucx = MappedSubsetTimeSeriesArrayField(
+        mapping=[{'source': 'ucx', 'subset': '2D_open_water'}])
+    ucy = MappedSubsetTimeSeriesArrayField(
+        mapping=[{'source': 'ucy', 'subset': '2D_open_water'}])
```
Comment (larsclaussen) [inline threedigrid/admin/nodes/timeseries_mixin.py]: imagine a source can be linked to two subsets, how would the mapping look like then?

File: threedigrid/orm/base/fields.py [hunk 2]
```diff
@@ -30,3 +31,63 @@ def __init__(self, to=None):
...
+    def get_mappings(self, value, model_instance):
+        SUBSETS = model_instance.SUBSETS
+
+        mappings = []
+
+        for subset_def in value:
+            name = subset_def['subset'].upper()
+            field_filter = [key for key in SUBSETS if
+                            name in SUBSETS[key]]
+            field_filter = field_filter[0]
```
Comment (larsclaussen) [inline threedigrid/orm/base/fields.py]: can field_filter be empty ``[]``?

File: threedigrid/orm/base/fields.py [hunk 3]
```diff
@@ -30,3 +31,63 @@ def __init__(self, to=None):
...
+
+        :param mapping: list of dicts with the mapping of fields like
+                        [{'source': 'ucx', 'subset': '2D_open_water'}]
+
+        :param null_value: null value to use outside the subset.
+        """
+        self.mapping = mapping
+        self.null_value = null_value
+
+    def get_mappings(self, value, model_instance):
```
Comment (larsclaussen) [inline threedigrid/orm/base/fields.py]: docstring is missing

File: threedigrid/admin/nodes/timeseries_mixin.py (hunk 1 above)
Comment (jpprins1) [reply threedigrid/admin/nodes/timeseries_mixin.py]: [{'source': 'ucy_2d_open_water', 'subset': '2D_open_water'}, {'source': 'ucy_1d_all', 'subset': '1D_all'}]

File: threedigrid/orm/base/fields.py (hunk 2 above)
Comment (jpprins1) [reply threedigrid/orm/base/fields.py]: Will add assert

**Pull request closed without merging**